src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -211,6 +211,8 @@ message PollOperationRequest {
 
 message BuildFarmServerConfig {
   InstanceConfig instance = 1;
+  
+  ActionCacheAccessPolicy ac_policy = 12;
```

Comment:
Is there a more standard (likely not protobuf, but maybe googleapis) type we could use for this?

Reply:
hmm,  [visibility](https://github.com/googleapis/googleapis/blob/master/google/api/visibility.proto#L89) is close.  We could consider the service method having a visibility restriction (i.e. not client facing).  
This [read-only / read-write mode](https://github.com/googleapis/googleapis/blob/master/google/datastore/v1/datastore.proto#L403) may be closer.  If we say the AC is a "datastore" and clients make "transactions" to it..

Reply:
landing for now, but I like the idea of adopting something from googleapi to represent our storages